WordPress/Sniffs/PHP/NoSilencedErrorsSniff.php

```diff
@@ -36,33 +36,35 @@ class NoSilencedErrorsSniff extends Sniff {
 	public $context_length = 6;
 
 	/**
-	 * Whether or not the `$function_whitelist` should be used.
+	 * Whether or not the `$allowedFunctionsList` should be used.
 	 *
 	 * Defaults to true.
 	 *
-	 * This property only affects whether the standard function whitelist is
-	 * used. The custom whitelist, if set, will always be respected.
+	 * This property only affects whether the standard function list is
+	 * used. The custom functions list, if set, will always be respected.
 	 *
 	 * @since 1.1.0
+	 * @since 3.0.0 Renamed from `$use_default_whitelist` to `$usePHPFunctionsList`.
 	 *
 	 * @var bool
 	 */
-	public $use_default_whitelist = true;
+	public $usePHPFunctionsList = true;
```

Comment:
Is anything going to flag this variable as not following snake_case formatting?

Reply:
No, we have disabled the [variable naming sniff for this repo](https://github.com/WordPress/WordPress-Coding-Standards/blob/fd44ba4300f8cbe73ee1b4c02547acb0664b3e66/.phpcs.xml.dist#L19) as PHPCS itself uses camelCase variables and we sometimes extend upstream sniffs and need to be able to change the default value of (camelCase) properties, so this will not be flagged.